Eatery Blue/Networking/EateryMemoryCache.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EateryMemoryCache.swift
+//  Eatery Blue
+//
+//  Created by Jayson Hahn on 10/25/23.
+//
+
+import Foundation
+import EateryModel
+
+actor EateryMemoryCache {
+    
+    
+    private var allLoaded = false
```

Comment:
```suggestion
    private var allLoaded: Bool = false
```
Keep styling consistent.

Reply:
Same idea i let type inference do its job

Reply:
The main reason is to keep it consistent with the other properties that you have below. I usually let type inference do the work but only if the other properties use it as well. Otherwise, keep it consistent and add the type.